pyclean/modern.py

```diff
@@ -78,7 +78,7 @@ def pyclean(args):
                   Runner.unlink_failed, Runner.rmdir_failed)
 
 
-def descend_and_clean_bytecode(directory):
+def descend_and_clean_bytecode(directory, clean_test_cache):
```

Comment:
There must be a more elegant way than mingling this option in here as an argument. :thinking: